build.sbt

```diff
@@ -30,6 +30,16 @@ lazy val migration = project
     libraryDependencies ++= Dependencies.Migration,
     publish / skip := true)
 
+lazy val migrator = project
+  .in(file("migrator"))
+  .enablePlugins(Publish)
+  .disablePlugins(SitePlugin, MimaPlugin)
+  .settings(
+    name := "akka-persistence-jdbc-migrator",
+    libraryDependencies ++= Dependencies.Migration ++ Dependencies.Libraries,
+    publish / skip := true)
```

Comment:
Add a TODO here so we don't forget to remove it when ready.

We want to publish the tool, but only when ready for use. It doesn't need to be when production ready. We may decide to publish it on a snapshot to test it out on real world projects.